azure-pipelines.yml

```diff
@@ -202,27 +207,28 @@ extends:
             }
           displayName: Test Deployed WebApp
 
-        - task: AzureCLI@2
-          displayName: Swap Staging Slot into Production
-          inputs:
-            azureSubscription: SourceDotNet-Deployment-ARM
-            scriptLocation: inlineScript
-            scriptType: ps
-            inlineScript: >
-              az webapp deployment slot swap
-              --resource-group source.dot.net
-              --name netsourceindex
-              --slot staging
-              --target-slot production
-
-        - task: AzureCLI@2
-          displayName: Cleanup Old Storage Containers
-          inputs:
-            azureSubscription: SourceDotNet-Deployment-ARM
-            scriptLocation: inlineScript
-            scriptType: ps
-            inlineScript: >
-              deployment/cleanup-old-containers.ps1
-              -ResourceGroup source.dot.net
-              -WebappName netsourceindex
-              -StorageAccountName netsourceindex
\ No newline at end of file
+        - ${{ if and(ne(variables['System.TeamProject'], 'public'), notin(variables['Build.Reason'], 'PullRequest'), eq(variables['Build.SourceBranch'], 'refs/heads/main')) }}:
```

Comment:
Consider adding a single variable for `publishToProduction` with the default value of `false` then use that to control both the slot name and these steps.